_posts/2020-12-05-data-quality-with-r.md

```diff
@@ -0,0 +1,2402 @@
+---
+layout: post
+title: An unexpected acquaintance? Data Validation in R
+author: filipwastberg
+excerpt: R is popular programming language and is famous for statistical analysis and stunning visualizations. But it is not limited to those tasks, here we take a look how it can be used to do data validation on a Snowflake database.
+date: 2020-12-17 06:30:00 +0100
+tags:
+ - R
+ - Data validation
+ - Snowflake
+ - ETL
+---
+Data Engineering is a common task for any Data Scientist. Sure, we
+rarely do the “real” Data Engineering, e.g. loading data from a
+business system into a relational database. Rather, we work with data 
+that already is in a database, and sometimes (if we're lucky) is put there by a Data Engineer. 
+We usually transform this data into an analytical format from which we can derive
+aggregations, visualizations and statistical models. In the start of any
+Data Science project these jobs may not be of any significant
+concern but as your Data Science products get used more and more the
+importance of your data will increase.
+
+Therefore it might be a good idea to invest some time into data
+validation. There are many frameworks for data validation in different
+programming languages. But Data Scientists usually feel comfortable
+using tools they know, such as R and Python. R is a popular functional
+programming language for data analysis. However, even though data
+analysis is what R is famous for, the language is turing complete and
```

Comment:
Maybe add a link to what does Turing complete mean, and some rationale to why is Turing complete important for a programming language?

Reply:
I think link is a good idea. I guess the idea is to show that R is not merely a programming language for statistics.